lib/services/northBound/restUtils.js

```diff
@@ -53,11 +53,9 @@ function checkMandatoryQueryParams(mandatoryAttributes, body, callback) {
     for (const p in mandatoryAttributes) {
```

Comment:
Entry to CNR needed.

Reply:
Done in https://github.com/telefonicaid/iotagent-node-lib/pull/1300/commits/eabcbf0c1f8c305e6b6d2d333254563ebb8f1713